stable/rbac-manager/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: rbac-manager
-version: 1.1.2
-appVersion: 0.6.1
+version: 1.1.3
```

Comment:
Since this is a minor version change to the underlying app, can you please make this a minor version change in the chart?